### PR TITLE
doc/Add breadcrumb info and make general edits

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

**Comment by PipKat** [comment]
@Andy-Grigg 
I'm sure my poor attempt to add breadcrumb information needs your attention!

File: doc/source/usage.rst [hunk 1]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
     html_css_files = ['css/custom.css']
 
-This way you can override the CSS style of this theme.
+Use your ``custom.css`` file to override the CSS style of this theme.
+
+Adding Breadcrumbs
+~~~~~~~~~~~~~~~~~~
+The PyAnsys Sphinx theme supports the display of breadcrumbs on
+the body of documentation pages to make navigation easier. To add
+breadcrumbs to the pages of your documentation, in the ``theme.conf``
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:63]:
```suggestion
breadcrumbs to the pages of your documentation, in the ``conf.py``
```

File: doc/source/usage.rst [hunk 2]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+file, set ``html_theme_option`` to ``"show_breadcrumbs": True``.
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:64]:
```suggestion
file, add ``"show_breadcrumbs": True`` to the ``html_theme_options`` dictionary.
```

File: doc/source/usage.rst [hunk 3]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+
+If you want to add additional 'root' breadcrumbs, such as to the 
+PyAnsys root page, you set ``html_theme_option`` to
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:67]:
```suggestion
PyAnsys Documentation homepage, add them to the ``html_theme_options`` dictionary as a list of tuples
```

File: doc/source/usage.rst [hunk 4]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+``"additional_breadcrumbs": [("Link text", "href")]``.
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:68]:
```suggestion
with the ``"additional_breadcrumbs"`` key. The tuples are of the form ``("Link text", "url")``.
```

File: doc/source/usage.rst [hunk 5]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+
+You must manually add the breadcrumb that links to the module homepage when on
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:70]:
```suggestion
When on the module homepage, a breadcrumb will be displayed with the homepage title.
```

File: doc/source/usage.rst [hunk 6]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+a page within the documentation. Because there is no way to get the title of the
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:71]:
```suggestion
However, this title is not accessible to Sphinx from other documentation pages. Therefore, the ``html_short_title``
```

File: doc/source/usage.rst [hunk 7]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+``index.rst`` page, you must use the ``html_title`` set in the ``conf.py``
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:72]:
```suggestion
is used as the display text for the documentation homepage breadcrumb. To ensure a consistent user experience
```

File: doc/source/usage.rst [hunk 8]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+file. The problem is that this isn't necessarily the same as the title of the
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:73]:
```suggestion
you should ensure the ``html_short_title`` (or optionally ``html_title`` if ``html_short_title`` is not used)
```

File: doc/source/usage.rst [hunk 9]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
+``index.rst`` page.
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:74]:
```suggestion
is set to the same value as the title of the ``index.rst`` page. If you want to use the version number
in the ``index.rst`` title, use ``|version|`` to include the package version number. 
```

File: doc/source/usage.rst [hunk 10]
```diff
@@ -47,11 +47,28 @@ file.  For example:
...
     html_static_path = ['_static']
     html_css_files = ['css/custom.css']
 
-This way you can override the CSS style of this theme.
+Use your ``custom.css`` file to override the CSS style of this theme.
+
+Adding Breadcrumbs
+~~~~~~~~~~~~~~~~~~
+The PyAnsys Sphinx theme supports the display of breadcrumbs on
+the body of documentation pages to make navigation easier. To add
```
Comment (Andy-Grigg) [inline doc/source/usage.rst:62]:
```suggestion
the body of documentation pages to make navigation easier. These 
breadcrumbs are disabled by default. To add
```

**Review by Andy-Grigg — CHANGES_REQUESTED** [review]
I have suggested some changes to the breadcrumb explanation. The other changes in this PR all make sense.

**Review by akaszynski — APPROVED** [review]
This looks good to me. Thanks for adding it!

**Pull request merged**